data/createFile.js

```diff
@@ -36,7 +37,7 @@ module.exports = async (arr, company) => {
     { semi: false, parser: 'babylon' }
   )
 
-  await fs.writeFile(`./data/single/${company}.js`, code, err => {
```

Comment:
This was causing issue when running in data folder instead of root

Reply:
Good thinking !